src/apps/common/submodules/appSelector/appSelector.scss

```diff
@@ -179,38 +183,82 @@
 			overflow-y: auto;
 			border-top: solid 1px $french-gray2;
 
-			.app-list {
+			ul.app-list {
 				margin-left: 0;
 
-				.app-item {
+				li {
 					box-sizing: border-box;
 					width: 100%;
-					background-color: $panel-background-color;
-					box-shadow: inset 0px 1px 0px 0px $french-gray2;
-					cursor: pointer;
-					display: flex;
-					flex-wrap: wrap;
-
-					&:hover {
-						background-color: $background-primary-color;
-					}
 
-					&.selected {
-						background-color: $french-pass;
+					&.app-item {
+						background-color: $panel-background-color;
+						cursor: pointer;
+						display: flex;
+						flex-wrap: wrap;
+						box-shadow: inset 0px 1px 0px 0px $white,
+							inset 0px -1px 0px 0px $white;
+
+						&:hover, &.selected {
+							background-color: $french-pass;
```

Comment:
Check the mock up again, the `:hover` color a slightly lighter the `.selected` one (use `lighten()`).

Reply:
I've missed it, I'll change the `:hover` color.